docker-compose.yml

```diff
@@ -1,7 +1,8 @@
-version: '2'
+version: '3'
 services:
   csgo:
-    image: 'dbaus/csgo:latest'
+    image: "dbaus/csgo:latest"
```

Comment:
Could you please leave this as single quotes?
Would like to have the convention along all the docker-compose file.

Thanks! :)